src/unified-mcp-server.ts

```
@@ -36,6 +36,12 @@ const __filename = fileURLToPath(import.meta.url);
 const __dirname = dirname(__filename);
 dotenv.config({ path: join(__dirname, '..', '.env.production') });
 
+// Global type declarations
+import type { Server } from '@modelcontextprotocol/sdk/server/index.js';
```

Biome (2.1.2)

[error] Shouldn't redeclare 'Server'. Consider to delete it or rename it.

'Server' is defined here:


(lint/suspicious/noRedeclare)